demo-shell/src/app/components/files/custom-sources.component.html

```diff
@@ -8,7 +8,7 @@
             </mat-select>
         </mat-form-field>
     </adf-toolbar-title>
-    <adf-toolbar-divider fxFlex="0 0 auto"></adf-toolbar-divider>
+    <adf-toolbar-divider></adf-toolbar-divider>
```

Comment:
I think this style was pushing all the buttons to the right, does it still do that?

Reply:
we discussed it on slack, it works as expected